### DISCUSSION TITLE
ChatGPT (or other AI Tools) [NEW RESOURCE]

### DISCUSSION THREAD

**Discussion by Nathan-Shea** [XargsUK/awesome-adhd#17] [Ideas]
## AI
Website and App

## Resource Description
I guess it kind of goes without saying. An AI tool which can be used as a personal assistant to fix some of your holes caused by ADHD.

## Resource URL
[https://chat.openai.com/auth/login]

## Why do you recommend it?
I've found this to be a godsend for me when it comes to 'uncluttering' my mind. I often need a skeleton framework to go off for lots of little tasks (like boring letters or emails) or they either won't get done or are an absolute mess. Also good for focusing my google searches down to something that will actually show results. Honestly this list could be 100 miles long. 

## Is there a cost associated with this resource?
Only if you want to use the API or ChatGPT 4

I haven't used it yet, but as an alternative there is [https://beta.character.ai/] it seems to contain more different tools etc.

**Comment by XargsUK** [comment]
First of all, massive thank you for the suggestion! 

I've been thinking quite a lot about this one, trying to work out why I feel that it wouldn't be a good fit to the repo; especially given that  I find ChatGPT super useful as well, and it's undeniable that a large percentage of those with ADHD have found it invaluable as a tool. I've tried to distill some of these thoughts below, and I'll leave it open in case other people have different thoughts, disagree, or so they can reference this issue <3 

ChatGPT and other AI tools are useful to both NT and ND minds, I think this is the primary strength of ChatGPT, as it's versatile across different applications. Introducing tools which are useful to both NT and ND brains could expand the scope of the repo, making it a little less targeted. 

I think this is why things like goblin.tools was added to the repo, yet ChatGPT was not. goblin.tools seems to target the specific need of breaking down large tasks into smaller tasks, which massively helps ND people with task management. While ChatGPT can also do this, it's extremely general in it's utilisation. 

A thought I had would be to either expand on the Wiki of this repo with different tips, which I've already done with a couple of things: 

- https://github.com/XargsUK/awesome-adhd/wiki/Apple-Watch-Tip
- https://github.com/XargsUK/awesome-adhd/wiki/Windows-11:-Clock

I could put together a Wiki of some ChatGPT prompts which are useful for those with ADHD, as this would be about guiding people with ADHD in how they can utlise a general tool in a way which works for ND minds.

**Reply by Nathan-Shea** [reply]
Slightly delayed reply!

All good! I completely get where you are coming from here, and goblin.tools is definitely a better tool that is specific to the issue at hand. At this point recommending ChatGPT is kind of like recommending Google with how general/broad the tool is. I would definitely be interested in a wiki section with some prompts etc though. 

Thanks again!
